application/forms.py

```diff
@@ -1,6 +1,5 @@
 import wtforms
 from wtforms import validators
-from classes.user import User
```

Comment:
Separate your imports into three categories: *standard*, *third party* and *local imports*